app/routers/cell_composition.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from fastapi import APIRouter, Response
+
+COMPOSITION_SUMMARY = (
+    (Path(__file__).parent.parent / "static/cellCompositionSummary_payload_prod.json").open().read()
```

Comment:
To avoid explicitly opening and closing the file, we can use:
```suggestion
    (Path(__file__).parent.parent / "static" / "cellCompositionSummary_payload_prod.json").read_text()
```
We could also json load/dump to be sure that the file contains valid json, but that's not so important since we control the file, and this is temporary.

Reply:
True, but I hope this goes away before we have problems like that.